packages/web/src/pages/App.module.css

```diff
@@ -58,12 +58,11 @@ body {
 }
 
 .mainContentWrapperMobile {
-  min-height: 100vh;
```

Comment:
I noticed we had some scrolling happening even when the content was smaller than the height, and this was the culprit.
_before_
![2023-11-15 13 05 03](https://github.com/AudiusProject/audius-protocol/assets/6711655/22c896fd-bd48-40e2-bcc4-b949166aa7bc)
_after_
![2023-11-15 13 06 11](https://github.com/AudiusProject/audius-protocol/assets/6711655/709dced6-1f04-4865-8b26-805cf6e1a18f)